src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -85,6 +85,10 @@
                  "aborted-completed",
                  "rejected"]
 
+ARCHIVED_STATUS = ["normal-archived",
```

Comment:
Given that we have a constant named `REQUEST_START_STATE`, maybe we could rename it to `REQUEST_END_STATES`?

Reply:
agreed